apps/storybook/src/ProfileSkills.stories.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { array } from '@storybook/addon-knobs';
-import { SkillsSection } from '@asap-hub/react-components';
+import { ProfileSkills } from '@asap-hub/react-components';
```

Comment:
Not sure about this name change. This component is used on Profile and Teams.

Reply:
Hmm, I wasn't aware of that.
What do you think of wording everything that is specific to users `UserProfile`, everything that is specific to teams `TeamProfile` and everything that is for both `Profile`?

Reply:
(in another PR ofc)